modules/primer-buttons/README.md

```diff
@@ -245,6 +245,21 @@ Use `.hidden-text-expander` to indicate and toggle hidden text.
 
 You can also make the expander appear inline by adding `.inline`.
 
+
+#### `<details>`/`<summary>` button styles
+
+You can apply `.btn` classes to `<summary>` elements, and they the
+selected/active styles will be applied when the parent `<details>` is open.
+
```

Comment:
Thinking about it, I'm not sure these docs fit here, I think they'd fit better with the details component when we add that module to Primer.